.github/workflows/bundle-app.yml

```diff
@@ -67,6 +68,7 @@ jobs:
         if: always()
         with:
           node-version: 14.x
+          cache: yarn
```

Comment:
Maybe yarn should be installed before this action runs?

Reply:
yup